Ocelot-Social-Community/Ocelot-Social
# refactor(database): update neo4j to 4.4

## 🍰 Pullrequest


This PR updates the neo4j database to 4.4

### Issues

- fixes https://github.com/Ocelot-Social-Community/Ocelot-Social/issues/6087
- based on https://github.com/Ocelot-Social-Community/Ocelot-Social/pull/6303

### Todo

- [ ] Investigate how this behaves on a kubernetes cluster

### How to test?

1. check current database contents
2. check out branch
3. `docker-compose up --build`
4. See all the old content
5. ...
6. Profit
